spec/linter-eslint-spec.js

```diff
@@ -367,24 +380,14 @@ describe('The eslint provider for Linter', () => {
     let editor
     let didError
     let gotLintingErrors
-    let tempFixtureDir
-    let tempFixturePath
+    const tempFixtureDir = fs.mkdtempSync(tmpdir() + path.sep)
```

Comment:
@Arcanemagus I ended up moving it to the function after all.  Seemed … cleaner, I guess.